### PR TITLE
fix: change runtime path to relative when ejecting runtime

### LINKED ISSUE #4066: [Settings] Runtime path should be relative
## Describe the bug
Runtime path should be a relative file path instead of an absolute file path. Otherwise, developers using github to cooperate with each other need to modify the runtime path everytime they pull.

## Version
latest

## Browser

- [ ] Electron distribution
- [x] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Edge

## OS

- [ ] macOS
- [x] Windows
- [ ] Ubuntu

## To Reproduce

1. Go to settings
2. Enable custom runtime
3. Check the settings, the file path is an absolute path

## Expected behavior

custom runtime is a relative path

## Screenshots
image

### PR DESCRIPTION
**Pull request by VanyLaw**
## Description

change path to relative when ejecting runtime

## Task Item

close #4066

### CODE REVIEW

File: extensions/runtimes/src/index.ts [hunk 1]
```diff
@@ -129,7 +129,9 @@ export default async (composer: any): Promise<void> => {
         await copyDir(schemaSrcPath, localDisk, schemaDstPath, project.fileStorage, pathsToExclude);
         const schemaFolderInRuntime = path.join(destPath, 'azurewebapp/Schemas');
         await removeDirAndFiles(schemaFolderInRuntime);
-        return destPath;
+        console.log(project.dir);
```
Comment (boydc2014) [inline extensions/runtimes/src/index.ts:132]: please remove the debugging message here

Comment (VanyLaw) [reply extensions/runtimes/src/index.ts]: done

**Comment by zidaneymar** [comment]
verified this could work in local echo ejected runtime (relative path)

**Comment by zidaneymar** [comment]
also work for azure publishing bot

**Review by luhan2017 — APPROVED** [review]

**Review by luhan2017 — APPROVED** [review]

**Review by a-b-r-o-w-n — APPROVED** [review]

**Pull request merged**